thread/src/components/RegisterForm.js

```diff
@@ -54,79 +78,98 @@ class RegisterForm extends Component {
         }
 
         return (
-            <Button full style={{ marginTop: 25 }} onPress={this.onSubmitPress}>
+            <Button rounded style={styles.buttonStyle} onPress={this.onSubmitPress}>
                 <Text>Register</Text>
             </Button>
         );
     }
 
     render() {
         return (
-            <Container style={{ backgroundColor: '#fff' }}>
+            <Container style={{ backgroundColor: '#8bc34a' }}>
                 <Content>
-                    <Form>
+                    <Form style={{padding:20, marginLeft: -20}}>
                         <Item stackedLabel>
-                            <Label>First Name</Label>
+
+                            <TextStyle>First Name</TextStyle>
                             <Input
+                                placeholder=''
+                                style={{fontSize:13}}
                                 onChangeText={this.onFirstNameChange}
                                 value={this.props.first_name}
                             />
+
                         </Item>
-                        <Item stackedLabel last>
-                            <Label>Last Name</Label>
+                        <Item style={{marginTop:20}} stackedLabel>
```

Comment:
This whole component is repeated, as well on the login page, for example make this a common component

```
<Item style={{marginTop:20}} stackedLabel>
    <TextStyle>Confirm Password</TextStyle>
    <Input
        style={{fontSize:13}}
        secureTextEntry
        onChangeText={this.onSecondPasswordChange.bind(this)}
        value={this.props.second_password}
    />
</Item>
```